packages/salesforcedx-utils/.eslintrc.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
These files were largely ported from the vscode-utils package. Things to keep an eye out for: 
- no imports of vscode.
- Note there are no tests.  I did not port the mocha tests from utils -> here.  I will write fresh jest tests as an example for this package.

Reply:
Verified! There are no imports from vscode.